pyrate/utils/__init__.py

```diff
@@ -10,7 +10,7 @@ def build_oauth1(client_key, client_secret, resource_owner_key, resource_owner_s
 
 def clean_dict(dirty_dict):
     """Cleans a dictionary from keys with empty string values"""
-    return dict((k, v) for k, v in dirty_dict.iteritems() if v)
+    return dict((k, v) for k, v in dirty_dict.items() if v)
```

Comment:
this unfortunately will copy the entire dict into memory in Py2 - could you please add six to the app and use [``six.iteritems``](Hey there, thanks for your PR!) instead? Thanks!